stim/rootcmd.go

```diff
@@ -28,6 +29,8 @@ func initRootCommand(viper *viper.Viper) *cobra.Command {
 	viper.BindPFlag("noprompt", cmd.PersistentFlags().Lookup("noprompt"))
 	cmd.PersistentFlags().StringP("auth-method", "", "", "Default authentication method (ex: ldap, github, etc.)")
 	viper.BindPFlag("auth.method", cmd.PersistentFlags().Lookup("auth-method"))
+	cmd.PersistentFlags().StringP("isautomated", "", "false", "Error on anything that needs to prompt and was not passed in as an ENV var or command flag")
```

Comment:
nit-pick but can we do `is-automated` to be consistent with the other vars?

Reply:
+1

Reply:
I originally had it that way but I was worried about how that translates into the config.yaml file.  The only other example of this is auth-method, which ends up as auth.method as the lookup which looks like:
```yaml
auth:
  method: "test"
```
in yaml.  I didnt like is-automated, translating to is.automated, which ends up in the config.yaml as:
```yaml
is:
  automated: true
```
so just moved to just is-automated.  I can have the --is-automated then just not translate the lookup in to the config the same way.  let me know what you guys think.

Reply:
That makes sense.  However, I don't think that dashes in parameters need to always directly translate into child elements in the config.  I could see us having some complex nesting in the config but want to simplify it for the command line parameters. With that said, I'm ok either way as we still need to clean up the config/arguments so we can address it then.